scripts/grpc/templates/client.usrv.cpp.jinja

```diff
@@ -44,7 +44,7 @@ constexpr const auto& k{{service.name}}ClientQosConfig =
 {{service.name}}Client::{{service.name}}Client(
     USERVER_NAMESPACE::ugrpc::client::impl::ClientParams&& client_params)
     : impl_(std::move(client_params), GetMetadata(),
-            std::in_place_type<{{proto.namespace}}::{{service.name}}>) {
+            std::in_place_type<::{{proto.namespace}}::{{service.name}}>) {
```

Comment:
Instead of adding `::`, try removing `{{proto.namespace}}::` in all of those places